gruntfile.js

```diff
@@ -122,6 +122,23 @@ module.exports = function(grunt) {
         },
 
         copy: {
+            modulescss: {
+                expand: true,
+                cwd: 'node_modules/',
+                src: ['d3fc/dist/d3fc.min.css', 'malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.css'],
+                dest: 'public/assets/css/',
+                flatten: true
```

Comment:
Would we rather import the CSS via the LESS file? (Maybe we should ask someone who knows more?)